numpy/distutils/command/build_ext.py

```diff
@@ -558,9 +558,10 @@ def _libs_with_msvc_and_fortran(self, fcompiler, c_libraries,
             # correct path when compiling in Cygwin but with normal Win
             # Python
             if dir.startswith('/usr/lib'):
-                s, o = exec_command(['cygpath', '-w', dir], use_tee=False)
-                if not s:
-                    dir = o
+                try:
+                    dir = subprocess.check_output(['cygpath', '-w', dir])
+                except subprocess.CalledProcessError:
```

Comment:
Are you sure you want this and not `OsError`, which is what is thrown if the command does not exist?

Reply:
(or perhaps both)

Reply:
And probably other things too:
```Python
import subprocess
ret = subprocess.check_output(['garbage', 'not', 'a', 'command'])
```

`FileNotFoundError: [Errno 2] No such file or directory: 'garbage': 'garbage'`

Would be much better if this was all well-tested, obviously.

Reply:
`FileNotFoundError` is a subclass of `OSError`, and doesn't exist at all on python 2